tests/TestCase/Routing/Route/RouteTest.php

```diff
@@ -20,6 +20,29 @@
 use Cake\TestSuite\TestCase;
 
 /**
+ * Used to expose protected methods for testing.
+ */
+class RouteProtected extends Route
+{
+    /**
+     * @return array
+     */
+    public function peekExtensions()
+    {
+        return $this->_extensions;
```

Comment:
I just wanted to test the setter function without being dependent upon the getter function. I can use the getter if you want, but I'm just being defensive with the tests.